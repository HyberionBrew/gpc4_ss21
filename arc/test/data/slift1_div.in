0: y = 1
1: z = 2
1: y = 2
2: y = 2
3: z = 2
3: y = 0
50: y = 2
51: z = 10
