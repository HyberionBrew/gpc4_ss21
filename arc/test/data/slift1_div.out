1: f = 1
2: f = 1
3: f = 1
50: f = 1
51: f = 5
